mappings/net/minecraft/block/Material.mapping

```diff
@@ -3,54 +3,76 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15912 breakByHand Z
 	FIELD field_15913 COBWEB Lnet/minecraft/class_3614;
 	FIELD field_15914 STONE Lnet/minecraft/class_3614;
+		COMMENT Hard solid nonmetallic mineral matter of which rock is made, especially as a building material.
```

Comment:
```suggestion
```
I don't think we need to describe the chemical composition of stone in Yarn javadocs (especially here).

Reply:
Rather than mentioning chemical composition, we may consider mentioning their breaking property (i.e. blocks that general can only be harvested with pickaxes and be broken faster with pickaxes)